src/impl/device/rotarysensor/rotation.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#include "okapi/impl/device/rotarysensor/rotation.hpp"
+
+namespace okapi {
+RotationSensor::RotationSensor(const std::uint8_t iport, const bool ireversed) {
+  port = iport;
+  pros::c::rotation_set_reversed(port, ireversed);
```

Comment:
Up to @Octogonapus , but maybe `reversed` should be an internal flag like with `Motor`, so that multiple instances of the class won't be interfering with the direction of the sensor.